src/components/scenegraph/Toolbar.js

```diff
@@ -41,8 +60,61 @@ export default class Toolbar extends React.Component {
     super(props);
 
     this.state = {
-      isPlaying: false
+      isPlaying: false,
+      objects: [],
+      linkToIdMap: null
```

Comment:
Is this going to be used in a diff PR? Like the asset dropdown? Because I couldn't find the asset dropdown on here when running it

Reply:
isPlaying is from before,
objects is for this PR
linkToIdMap is to go from the asset link to asset id, partially used for this PR, since this PR also handles updating the asset